src/components/ResourceCard/AddResource/ResourceDescribeActivity.js

```diff
@@ -301,8 +278,9 @@ ResourceDescribeActivity = reduxForm({
 const mapDispatchToProps = (dispatch) => ({
   showBuildActivity: (editor, editorType) => dispatch(showBuildActivityAction(editor, editorType)),
   onSubmitDescribeActivity: (metadata) => dispatch(onSubmitDescribeActivityAction(metadata)),
-  uploadResourceThumbnail: (formData) => dispatch(uploadResourceThumbnailAction(formData)),
+  uploadResourceThumbnailAction: (formData) => dispatch(uploadResourceThumbnailAction(formData)),
   goBackToActivity: () => dispatch(showSelectActivity()),
+  uploadResourceThumbnail: (url) => dispatch(uploadResourceThumbnail(url)),
```

Comment:
Why do we need 2 actions for upload thumbnail?
Please check how it is done on project thumbnail

Reply:
one is for google pexel and second if for upload image from local